lib/tern.js

```diff
@@ -749,8 +749,14 @@
       if (expr) return expr;
       expr = infer.findExpressionAround(file.ast, start, end, file.scope);
       if (expr && (expr.node.type == "ObjectExpression" || wide ||
-                   (start == null ? end : start) - expr.node.start < 20 || expr.node.end - end < 20))
+                   (start == null ? end : start) - expr.node.start < 20 || expr.node.end - end < 20)) {
+        if (expr.node.type == "ObjectExpression" ) {
+          // return object literal property if possible.
+          var objProp = pointInProp(expr.node, end);
+          if (objProp) return {node: objProp, state: expr.state};          
```

Comment:
I don't think it's a good idea for `findExpr` to return a non-expression (as your special casing further down in the patch shows, it causes problems). Would it make sense to move this test (whether the expr is an object's property) into `findTypeAt` instead?